experiments/langchain_lab/chatbot_demo/__init__.py

```diff
@@ -0,0 +1,59 @@
+"""Samples of Chatbot."""
```

Comment:
Should the script be put in another file not named with __init__.py? 
Because from my knowledge as far, I thought __init__.py only controls imports of the subfolder/subpackage, and function def were usually defined in another module.

Reply:
Your understanding is correct. Here I just want to save the length of import while I could figure out a better name to hold the functions `get_rag_chatbot`. But thanks for asking and after a second thought, I decided to rename it as `rag.py`.

Reply:
thanks for the updating! Now I can see rag.py. Btw, the `get_rag_chatbot` in __init__.py is duplicated with `get_chatbot` in rag.py.

Reply:
Thanks for the reminding. Removed.